tests/functional/views/test_legacy_publishing.py

```diff
@@ -1,366 +1,432 @@
+import unittest
+from pyramid import testing
 from lxml import etree
 
 from litezip.main import COLLECTION_NSMAP
 
+class TestBypassingAuth():
```

Comment:
The naming confused me. You aren't by-passing authentication or authorization in this test suite. You are actually using it.